plugins/woocommerce-blocks/assets/js/base/components/reviews/review-list-item/index.tsx

```diff
@@ -159,6 +172,9 @@ const ReviewListItem = ( { attributes, review = {} }: ReviewListItemProps ) => {
 	const { rating } = review;
 	const isLoading = ! ( Object.keys( review ).length > 0 );
 	const showReviewRating = Number.isFinite( rating ) && showReviewRatingAttr;
+	const reviewRatingId = `rating-stars-${ Math.floor(
+		Math.random() * 100
```

Comment:
You can use [`useId`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-element/#useid) here

Reply:
Done!
Thanks for the help.
- [Use useId hook](https://github.com/woocommerce/woocommerce/pull/54811/commits/c0e435582f8c3b71ca0dbe26f150a93ae22226e8)